src/network/server.cc

```diff
@@ -140,6 +141,79 @@ void NetworkServer::getBlobData(PLocalObject target_object, QTcpSocket *client_c
   sendData(client_connection, reinterpret_cast<const char*>(blob->data().rawData()), blob->data().octets());
 }
 
+void NetworkServer::addChunkItem(PLocalObject blob, network::Response &resp,
+                                 PLocalObject object,
+                                 const network::ChunkDataItem &req_item) {
+  if (object->type() != dbif::CHUNK) {
+    resp.set_ok(false);
+    resp.set_error_msg("Items can only be added to chunk");
+  }
+  auto chunk = object.staticCast<ChunkObject>();
+  if (req_item.repack().endian() >= static_cast<int>(data::RepackEndian::GUARD)) {
```

Comment:
I wonder if you can mess up some of this checks by supplying very large value on packet (to flip the int)?